fulp_modules/main_features/bloodsuckers/code/powers/masquerade.dm

```diff
@@ -31,15 +31,12 @@
 	return TRUE
 */
 
-/datum/action/bloodsucker/masquerade/ActivatePower()
-
-	var/mob/living/user = owner
+/datum/action/bloodsucker/masquerade/ActivatePower(mob/living/carbon/user = owner)
```

Comment:
This has to be mob/living/carbon because otherwise the getorganslot just empty returns on us